Source/WebCore/page/LocalFrameView.h

```diff
@@ -1073,6 +1073,6 @@ WTF::TextStream& operator<<(WTF::TextStream&, const LocalFrameView&);
 } // namespace WebCore
 
 SPECIALIZE_TYPE_TRAITS_BEGIN(WebCore::LocalFrameView)
-static bool isType(const WebCore::AbstractFrameView& view) { return view.viewType() == WebCore::AbstractFrameView::FrameViewType::Local; }
+static bool isType(const WebCore::FrameView& view) { return view.viewType() == WebCore::FrameView::FrameViewType::Local; }
```

Comment:
When you see this identifier here it seems pretty clear it should be `FrameView::Type` not `FrameView::FrameViewType`.

Reply:
Indeed, fixing. Thanks.